### PR TITLE
fix acs url in google saml sso docs

### PR DESCRIPTION
**Pull request by LalitDeore**

### CODE REVIEW

File: docs/extensions.md [hunk 1]
```diff
@@ -76,7 +76,9 @@ After adding them, click "Save", saving the configuration. After saving, log out
 
 • Next page will be the SP information, this is where you should provide the Single Sign On URL, and SP Entity ID.
 
-	- ACS URL : https://10.67.0.70:3443/api/v1/login_sso
+	- ACS URL : 
+ 		- For Cloud : https://shuffler.io/api/v1/login_sso
+		- For On-prem: https://<your-ip>:3443/api/v1/login_sso	
 	- Entity ID : shuffle-saml
```
Comment (frikky) [inline docs/extensions.md:82]: Did you read the email I sent? They were confused by the Entity ID. Does the Entity ID matter?

Please answer them as well.

Comment (frikky) [inline docs/extensions.md:82]: Did you read the email I sent? They were confused by the Entity ID. Does the Entity ID matter?

Please answer them :)

Comment (LalitDeore) [reply docs/extensions.md:82]: I test it by changing the entity id and it work for me no error. I will answer them.

for documentation, I am gonna create new issue just for improving the sso docs for the shuffle. In future we can improve the overall sso docs.

Comment (frikky) [reply docs/extensions.md:82]: Good plan - thanks :)

**Pull request merged**